app/views/dashboards/show.html.erb

```diff
@@ -49,7 +49,7 @@
           <% @presenter.collections.each do |collection| %>
             <li>
               <div>
-                <header><%= collection.name %>
+                <header><%= collection.name.presence || 'No Title' %>
```

Comment:
Now that we allow draft collections, they might not have a title (similar to the works, which do something like this too for display purposes on the dashboard).